src/Fantomas/CodePrinter.fs

```diff
@@ -2641,10 +2641,20 @@ and genExpr astContext synExpr ctx =
         | IndexRangeExpr (None, None) -> !- "*"
         | IndexRangeExpr (Some (IndexRangeExpr (Some (ConstNumberExpr e1), Some (ConstNumberExpr e2))),
                           Some (ConstNumberExpr e3)) ->
+            let hasOmittedTrailingZero =
+                ctx.Content[e1.Range.EndColumn] = '.'
```

Comment:
This is quite unsafe to use. Content will only be present when there was source code.
So, it won't be there when formatting from AST for example.
Also, I believe content is one string, so you got a bit lucky with the `Range.EndColumn`, which will probably only work for the first line.

Reply:
Thanks for the review. I had some bad misunderstanding about ctx.Content in my head.
I'll try to come up with something over the next couple of days.